Parsers/KVPairExtraction

```diff
@@ -0,0 +1,5 @@
+| extend kvpairs=parse_json(extract_all("([^\\d\\s]+)=(\\S+),\\s", dynamic([1,2]), SyslogMessage))
```

Comment:
Please follow the parser format identified here - https://github.com/Azure/Azure-Sentinel/wiki/Contribute-to-Sentinel-GitHub-Community-of-Queries#example-parser